wallet-api/src/Wallet/Generators.hs

```diff
@@ -178,9 +180,31 @@ genValidTransactionSpending' g f ins totalVal = do
                     txValidRange = $$(Interval.always) }
         else Gen.discard
 
+genAda :: MonadGen m => m Ada
+genAda = Ada.fromInt <$> Gen.int (Range.linear 0 (100000 :: Int))
+
+genValue' :: MonadGen m => Range Int -> m Value
+genValue' valueRange = do
+    let currencyRange = Range.linearBounded @Int
+        pair          = (,) <$> Gen.int currencyRange <*> Gen.int valueRange
+        sngl          = uncurry Value.singleton <$> pair
```

Comment:
`sngl = Value.singleton <$> Gen.int currencyRange <*> Gen.int valueRange`?

Reply:
Ah yes, thanks, this was the result of some earlier simplifications